my-project/src/app/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import LoginForm from '../components/LoginForm/index'
+import './style.scss';
+
+class App extends Component {
+    submit = values => {
+        alert(JSON.stringify(values));
+    };
+    render() {
+        return (
+            <>
+                <h1 className="title">Sing in</h1>
+                <LoginForm onSubmit={this.submit}></LoginForm>
```

Comment:
It isn't necessary closing tag here:
`<LoginForm onSubmit={this.submit} />`

Reply:
done